api/v1/v1.go

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2017 The Decred developers
+// Copyright (c) 2019 The Decred developers
```

Comment:
You don't replace the copyright. You add a year. Like: `2017-2019`

Reply:
Thanks, fixed.